Cisco/cisco-esa/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,291 @@
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "DROPPED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was dropped"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was bounced",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "BOUNCED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was bounced"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was aborted",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "ABORTED"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was aborted"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was DQ",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "DQ"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was DQ"
+            }
+        ]
+    },
+    {
+        "value": "Email from {email.from.address} to {email.to.address} was delivered",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "blocked"
+            },
+            {
+                "field": "email.from.address"
+            },
+            {
+                "field": "email.to.address"
+            }
+        ],
+        "relationships": [
+            {
+                "source": "email.from.address",
+                "target": "email.to.address",
+                "type": "message was blocked"
+            }
+        ]
+    },
```

Comment:
The description mismatch the value of `event.action`.
Are you sure that `blocked` is a value for `event.action`?